kube/ingress.yml

```diff
@@ -2,20 +2,24 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
+  labels:
+    stable.k8s.psg.io/kcm.class: default
   annotations:
-    ingress.kubernetes.io/secure-backends: "true"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: "nginx-external"
+    stable.k8s.psg.io/kcm.provider: http
   name: developer-induction-ingress
 spec:
   rules:
-  - host: developer-induction.notprod.homeoffice.gov.uk
-    http:
-      paths:
-      - backend:
-          serviceName: developer-induction-service
-          servicePort: https
-        path: /
+    - host: developer-induction.notprod.homeoffice.gov.uk
```

Comment:
Do you really want to deploy on this subdomain? Zero chance of getting a cert with notprod.